docs/whats-new.md

```diff
@@ -2,6 +2,54 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.2
+
+Release Data: TBD, 2020
+
+(Need screenshots: globe view, pattern fill, MVT cross-tile highlight)
+
+### Tiled Layers
+
+Many new features are added to `TileLayer`, `MVTLayer` and `TerrainLayer` to improve correctness, performance and ease of use.
+
+- `MVTLayer` projects more accurately at high zoom levels.
+- `autoHighlight` in the `MVTLayer` now works on features that are split across multiple tiles, identified by the new `uniqueIdProperty` prop.
+- All geospatially tiled layers now handle high pitch angles better. Far away tiles from the camera are loaded at lower zoom levels, avoiding loading too many tiles.
+- Tiled layers now use a request scheduler to prioritize loading the most recently visible tiles during viewport navigation. See the new `throttleRequests` and `maxRequests` props.
+- `TileLayer` may continue to display tiles when underzoomed. See the new `extent` prop.
```

Comment:
From the API docs

> If provided, the layer will load and render the tiles in this box when underzoomed

I can't tell if that means _new_ tiles will also be loaded or only that tiles already loaded will continue to be displayed. Could be good to clarify, maybe in a separate PR.